File: python/telemetry-api-demo/handler/handler.py
def handler(event, context):
    print(f'🐚🐚🐚🐚🐚 got event: {event} 🐚🐚🐚🐚🐚')
    print(f'🐚🐚🐚🐚🐚 got context: {context} 🐚🐚🐚🐚🐚')
    return {'Hello': 'World!'}

File: java/lambda/src/main/java/example/Handler.java
package example;

import com.amazonaws.services.lambda.runtime.Context;
import com.amazonaws.services.lambda.runtime.RequestHandler;
import com.amazonaws.services.lambda.runtime.events.APIGatewayV2HTTPEvent;
import com.amazonaws.services.lambda.runtime.events.APIGatewayV2HTTPResponse;

// Handler value: example.Handler
public class Handler implements RequestHandler<APIGatewayV2HTTPEvent, APIGatewayV2HTTPResponse>{

  @Override
  public APIGatewayV2HTTPResponse handleRequest(APIGatewayV2HTTPEvent event, Context context)
  {
    APIGatewayV2HTTPResponse response = new APIGatewayV2HTTPResponse();
    response.setStatusCode(200);
    response.setBody("{\"hello\":\"world\"}");
    return response;
  }
}
